package.json

```diff
@@ -10,22 +10,22 @@
     "chalk": "^1.1.1",
     "cli-table": "^0.3.1",
     "https-proxy-agent": "^1.0.0",
-    "joi": "^6.9.1",
-    "nodesecurity-npm-utils": "^4.0.1",
+    "joi": "^6.10.1",
+    "nodesecurity-npm-utils": "^5.0.0",
     "path-is-absolute": "^1.0.0",
     "rc": "^1.1.2",
     "semver": "^5.0.3",
     "subcommand": "^2.0.3",
     "wreck": "^6.3.0"
   },
   "devDependencies": {
-    "code": "^1.5.0",
+    "code": "^3.0.1",
     "eslint": "^2.5.3",
     "eslint-config-nodesecurity": "^1.3.1",
-    "eslint-plugin-hapi": "^1.2.2",
+    "eslint-plugin-hapi": "^4.0.0",
     "git-validate": "^2.1.0",
-    "lab": "^6.2.0",
-    "nock": "^7.7.2",
+    "lab": "^10.8.2",
```

Comment:
Lab was at 6 specifically because that's where the cutoff on node 0.12 was. If we want to test this in 0.12 we have to stick with lab 6